### PR TITLE
Disable polyfill on 3.25+

### PR DESCRIPTION
**Pull request by chancancode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,7 @@ ember-named-blocks-polyfill
 ==============================================================================
 
 This addon provides a polyfill for the [Yieldable Named Blocks][RFC] feature.
-When the feature lands in Ember.js, this addon will be updated to be inert on
-versions with native support.
+On Ember.js versions with native support for the feature, this addon is inert.
```
Comment (rwjblue) [inline README.md:5]: Might want to say 3.25.0-beta.1 or higher specifically.

File: config/ember-try.js [hunk 2]
```diff
@@ -23,34 +23,31 @@ module.exports = async function() {
         }
       },
       {
-        name: 'ember-release',
+        name: 'ember-lts-3.20',
         npm: {
           devDependencies: {
-            'ember-source': await getChannelURL('release')
+            'ember-source': '~3.16.1'
```
Comment (rwjblue) [inline config/ember-try.js:29]:
```suggestion
            'ember-source': '~3.20.0'
```

**Review by rwjblue — APPROVED** [review]

**Comment by bartocc** [comment]
I thought that the deviations previously listed in `Readme.md` would not be present in the final implementation in `ember-source`.

For instance, I thought that

```hbs
<XFoo>
  <:row>row 1</:row>
  <:row>row 2</:row>
</XFoo>
```
 would be possible in the final implementation.

But in the end, these limitations will stay, right?

**Comment by rwjblue** [comment]
> But in the end, these limitations will stay, right?

Yes

**Comment by bartocc** [comment]
was that intended in the RFC or did you hit implementation issues and had to diverge from the RFC?

**Comment by rwjblue** [comment]
Unless I'm mistaken, we have implemented exactly what was in the RFC (at least in terms of your original question around having two named blocks with the same name).

🤔

**Comment by bartocc** [comment]
I probably got confused by the previous `Readme.md` and its `Deviations` section then 🤔

**Pull request merged**